packages/connectivity/src/scp-cf/authorization-header.ts

```diff
@@ -133,6 +133,9 @@ interface AuthenticationHeaders {
   'Proxy-Authorization'?: string;
   'SAP-Connectivity-Authentication'?: string;
 }
+interface AuthenticationHeaderSAMLAssertion {
+  'x-sap-security-session': string;
```

Comment:
[nth] This could be more explicit:
```suggestion
  'x-sap-security-session': 'create';
```